projects/bp-gallery/src/components/views/start/StartView.tsx

```diff
@@ -134,7 +139,11 @@ const StartView = () => {
         </div>
 
         <ShowStats>
-          <OverviewContainer defaultValue={1} tabs={tabs} />
+          <OverviewContainer
+            defaultValue={tabs.length > tabMode ? tabMode : 0}
```

Comment:
I was a bit confused by this ternary until I understood that it's a range check and `tabMode` is the default tab index. Maybe rename the feature to something like `start_view_default_tab_index`? Also, I don't know if it's just me, but I usually put range checks the other way around: `tabMode < tabs.length ? tabMode : 0`, I think it's more readable (or maybe I'm just weird :D).